docs/consumer/Converter.md

```diff
@@ -0,0 +1,114 @@
+# Stake Converter
+
+The Stake Converter is on the consumer side and is connected to an External Staker on the Provider side.
+This handles the normalization of the external tokens and _converts_ them into "Virtual Stake".
+There is a 1:1 connection between a Converter and a [Virtual Staking Contract](./VirtualStaking.md)
+which handles the actual issuance.
+The converter is connected to the Provider chain via IBC and handles the various packets coming from it.
+
+**TODO** This was called the "Consumer contract" in previous versions and probably needs a better name
```

Comment:
```suggestion
```

Can probably get rid of this. I'm ok with `mesh-converter` or `mesh-receiver`, leaning towards the later personally but not worth bike-shedding over.